lib/wax.ex

```diff
@@ -218,7 +218,8 @@ defmodule Wax do
 
   The returned structure:
   - Contains the challenge bytes under the `bytes` key (e.g.: `challenge.bytes`). This is a
-  random value that must be used by the javascript WebAuthn call
+  random value that must be used by the javascript WebAuthn call. Optionally, you can supply
+  your own via the `:bytes` option.
```

Comment:
We should then document it in 2 places:
- https://github.com/tanguilp/wax/blob/master/README.md#options
- https://github.com/tanguilp/wax/blob/master/lib/wax.ex#L16

Reply:
Thanks - updated in [6e2e95d](https://github.com/tanguilp/wax/pull/47/commits/6e2e95d1aad45cd6c0202787e56f952e3c0c7938)

Reply:
I've updated the messaging. Please provide feedback